clients/unshaded/src/main/java/tachyon/client/UnderStorageType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client;
+
+/**
+ * Specifies the type of data interaction with Tachyon's Under Storage. This is not applicable
+ * for reads.
+ */
+public enum UnderStorageType {
```

Comment:
Bin's comment suggests that the type does not really describe UnderStorage, it describes the type of the write operation.

What is the argument against simply using WriteType? For instance:

``public enum WriteType {
  PERSISTENT(1),
  NON_PERSISTENT(2);

  private final int mValue;
  WriteType(int value) {
    mValue = value;
  }

  public boolean isPersistent() {
    return mValue == PERSISTENT.mValue;
  }
}
``